cookbook/bundles/extension.rst

```diff
@@ -15,7 +15,7 @@ Creating an Extension Class
 ---------------------------
 
 In order to load service configuration, you have to create a Dependency
-Injection Extension for your bundle. This class has some conventions in order
+Injection (DI) Extension for your bundle. This class has some conventions in order
```

Comment:
A good practice when using acronyms is to introduce them properly before the first time they are used it. In this article, we talk about `Dependency Injection` and suddenly, the article talks about `DI`. I know that Symfony developers are smart enough to infere that `DI` means `Dependency Injection`, but the usual good practice is to show the full words + its acronym just before starting to use the acronym exclusively.